src/averageAges.js

```diff
@@ -14,18 +14,30 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  function getMenOfTheCentury() {
+    return people.filter(person =>
+      Math.ceil(person.died / 100) === century && person.sex === 'm');
+  }
+
+  function getAllMen() {
+    return people.filter(person =>
+      person.sex === 'm');
```

Comment:
Краще не розривати колбеки
```suggestion
    return people.filter(
      person => person.sex === 'm'
    );
```

Reply:
done